crates/anvil-polkadot/src/api_server/server.rs

```diff
@@ -42,17 +47,21 @@ use polkadot_sdk::{
     parachains_common::{AccountId, Hash, Nonce},
     sc_client_api::HeaderBackend,
     sp_api::{Metadata, ProvideRuntimeApi},
-    sp_core::{self, H160, H256, Hasher, keccak_256},
+    sp_arithmetic::Permill,
+    sp_core::{self, Hasher, keccak_256},
     sp_runtime::traits::BlakeTwo256,
 };
 use sqlx::sqlite::SqlitePoolOptions;
-use std::{sync::Arc, time::Duration};
+use std::{collections::HashSet, sync::Arc, time::Duration};
 use substrate_runtime::Balance;
 use subxt::{
     Metadata as SubxtMetadata, OnlineClient, backend::rpc::RpcClient,
-    client::RuntimeVersion as SubxtRuntimeVersion, ext::subxt_rpcs::LegacyRpcMethods,
+    client::RuntimeVersion as SubxtRuntimeVersion, config::substrate::H256,
+    ext::subxt_rpcs::LegacyRpcMethods, utils::H160,
 };
 
+pub const CLIENT_VERSION: &str = concat!("anvil/v", env!("CARGO_PKG_VERSION"));
```

Comment:
Not sure, I would name it `anvil-polkadot` to differentiate.

```suggestion
pub const CLIENT_VERSION: &str = concat!("anvil-polkadot/v", env!("CARGO_PKG_VERSION"));
```